### PR TITLE
Add unit test and integration test for compareModel

### PR DESCRIPTION
**Pull request by jshum2479**
There are some logic in the operator code depends on WDT behavior to report errors or changing update behavior, this is the first series of tests added to ensure future changes does not impact WKO.

### CODE REVIEW

**Review by ddsharpe — APPROVED** [review]

File: system-test/src/test/java/oracle/weblogic/deploy/integration/BaseTest.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
     private static String projectRoot = "";
```
Comment (rakillen) [inline system-test/src/test/java/oracle/weblogic/deploy/integration/BaseTest.java:30]: Needs a copyright update at the top.

File: system-test/src/test/java/oracle/weblogic/deploy/integration/ITWdt.java [hunk 2]
```diff
@@ -773,6 +773,31 @@ public void testNValidateModelWithoutVariableFile() throws Exception {
         logTestEnd(testMethodName);
     }
 
+    /**
```
Comment (rakillen) [inline system-test/src/test/java/oracle/weblogic/deploy/integration/ITWdt.java:776]: Needs a copyright update at the top.

Comment (jshum2479) [reply system-test/src/test/java/oracle/weblogic/deploy/integration/ITWdt.java:776]: done

File: system-test/src/test/java/oracle/weblogic/deploy/integration/BaseTest.java (hunk 1 above)
Comment (jshum2479) [reply system-test/src/test/java/oracle/weblogic/deploy/integration/BaseTest.java:30]: done

**Review by rakillen — APPROVED** [review]

**Pull request merged**